extensions/org.wso2.carbon.config.maven.plugin/src/main/java/org/wso2/carbon/config/maven/plugin/ConfigDocumentMojo.java

```diff
@@ -124,15 +145,24 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                             .description()));
                     // add root node to the config Map
                     finalMap.put(configuration.namespace(), readConfigurationElements(configObject, Boolean.TRUE));
+                    contentMap.put(configuration.namespace(), readConfigurationElements(configObject, Boolean.TRUE));
```

Comment:
Why are we need to call readConfigurationElement method again here. and why we need another map(contentMap). can't we use the same map.